packages/taraxa-ui/src/components/Card/index.ts

```diff
@@ -1 +1,3 @@
-export { default } from './Card';
+import Card from './Card';
```

Comment:
```suggestion
export * from './Card';
```

Can we export like this pls?